ibis/backends/sqlite/registry.py

```diff
@@ -19,32 +21,57 @@
 operation_registry.update(sqlalchemy_window_functions_registry)
 
 
+sqlite_cast = Dispatcher("sqlite_cast")
+
+
+@sqlite_cast.register(AlchemyExprTranslator, ir.IntegerValue, dt.Timestamp)
+def _(t, arg, _):
```

Comment:
Can we have a short function name here, like `int_to_timestamp`? Can make it easier to understand the actual overload.